lib/modules/dosomething/dosomething_rogue/dosomething_rogue.cron.inc

```diff
@@ -56,9 +56,9 @@ function dosomething_rogue_retry_failed_reportbacks() {
       } else {
         $data = [
           [
-            'rogue_reportback_item_id' => $task->rogue_item_id,
+            'rogue_event_id' => $task->rogue_event_id,
             'status' => $task->status,
-            'reviewer' => $task->reviewer,
+            'event_type' => 'post_photo',
```

Comment:
I can update that here! I think you both are right because on the Rogue side, for these updates, I've changed the `submission_type` to `admin review` instead of `user`. This will make more sense with both these changes.